public/javascripts/webrtc.js

```diff
@@ -407,14 +378,22 @@ const chatform = document.querySelector('#data');
     e.preventDefault();
     const form = e.target;
     const input = form.querySelector('#message');
+    const username = sessionStorage.getItem('name');
     const message = input.value;
 
+    appendUsername('peer', username,);
+    for(let peerID in $peers[TEXT_CHAT]) {
+      console.log('Username Displaying To: ', peerID);
+      $peers[TEXT_CHAT][peerID].dataChannel.send(username);
```

Comment:
Instead of sending this separately, I would suggest send the id together with the message (see the comment below)